src/lib/components/GeoJSON.react.js

```diff
@@ -150,7 +150,7 @@ GeoJSON.propTypes = {
     /**
      * Function that determines how a cluster is drawn.
      */
-    clusterToLayer: PropTypes.string,
```

Comment:
Ah, yes that is a bug. It can also be a string though, so the correct type os "PropTypes.oneOfType([PropTypes.string, PropTypes.object])"

Reply:
Ohh OK! I will change that.

Reply:
About the string, I wonder what is the syntax to use with this property. I tried to pass the full namespace (e.g. "dlx.scatter.clusterToLayer") but it doesn't work. Do you have any insight?

Reply:
It can be used in a few ways, one of the full path to the function, yes. I just updated the documentaion with a new examples,

http://dash-leaflet.herokuapp.com/#scatter_cluster

NB: The `dlx` namespace has been dropped in the most recent versions of dash-leaflet.